Running in wrapper mode for '296'

Running python job.

[name: "CPU-QUOTA"
int_val: 38
int_val: 7
int_val: 57
int_val: 16
int_val: 41
int_val: 71
int_val: 84
, name: "DISK"
int_val: 18
int_val: 27
int_val: 26
int_val: 11
int_val: 23
int_val: 28
int_val: 25
, name: "MEMORY"
int_val: 26
int_val: 38
int_val: 32
int_val: 39
int_val: 14
int_val: 33
int_val: 22
, name: "NET"
int_val: 28
int_val: 40
int_val: 10
int_val: 20
int_val: 20
int_val: 29
int_val: 23
]
The paramters are {u'CPU-QUOTA': array([38,  7, 57, 16, 41, 71, 84]), u'NET': array([28, 40, 10, 20, 20, 29, 23]), u'DISK': array([18, 27, 26, 11, 23, 28, 25]), u'MEMORY': array([26, 38, 32, 39, 14, 33, 22])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 17500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 204800.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 22649241.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1425000000.0
Recovering the memory utilization
Setting cpu quota to 95000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 143360.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 15099494.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 975000000.0
Recovering the memory utilization
Setting cpu quota to 142500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 51200.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 21810380.8
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 1200000000.0
Recovering the memory utilization
Setting cpu quota to 95000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 143360.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 15099494.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 975000000.0
Recovering the memory utilization
Setting cpu quota to 142500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 51200.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 21810380.8
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 1200000000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f4038499488>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.37 seconds.
